parent/pom.xml

```diff
@@ -107,8 +107,8 @@
     <proton-j-version>0.31.0</proton-j-version>
     <qpid-proton-j-extensions-version>1.2.0</qpid-proton-j-extensions-version>
     <log4j-api.version>2.11.1</log4j-api.version>
-    <groovy-eclipse-compiler.version>2.9.3-01</groovy-eclipse-compiler.version>
-    <groovy-eclipse-batch.version>2.4.16-03</groovy-eclipse-batch.version>
+    <groovy-eclipse-compiler.version>3.4.0-01</groovy-eclipse-compiler.version>
```

Comment:
Make sure nothing else is using this dependency

Reply:
Sure. The plugin is intended the run Java and Groovy code together in maven, I believe no other client using the plugin for now.

Reply:
Perfect, just wanted to make sure we weren't accidentally bumping a dependency on other libraries.